webknossos-datastore/app/com/scalableminds/webknossos/datastore/helpers/DatasetDeleter.scala

```diff
@@ -31,23 +39,244 @@ trait DatasetDeleter extends LazyLogging with DirectoryConstants {
         case e: Exception                                => Fox.failure(s"Deleting dataset failed: ${e.toString}", Full(e))
       }
 
+    def moveToTrash(organizationId: String,
```

Comment:
we should probably make everything in here either private or protected, unless it’s really necessary for it to be public.

Reply:
this is not private because it is defined in the deleteOnDisk method, apart from deleteOnDisk there are no non-private methods here.